src/bridge/mod.rs

```diff
@@ -91,7 +91,7 @@ fn build_nvim_cmd() -> Command {
     }
 }
 
-pub fn build_neovide_command(channel: u64, num_args: u64, command: &str, event: &str) -> String {
+pub fn _build_neovide_command(channel: u64, num_args: u64, command: &str, event: &str) -> String {
```

Comment:
Similarly here. I would double check though, its possible that this just isn't used for this platform but is used on a different platform. In which case you'd get the warning on some builds but not others. Not sure how to deal with that issue

Reply:
Yep its usage was guarded under `#[cfg(windows)]` so I'll change it back

Reply:
Guess I should not rely on lsp references